### DISCUSSION TITLE
Small issue with getting this working on Windows

### DISCUSSION THREAD

**Discussion by ClayPippenger** [arc53/DocsGPT#5] [General]
I was not able to get the sample working on Windows due to this error:

![image1](https://user-images.githubusercontent.com/41549061/216796810-436cf2c1-a0d4-4f10-bfa3-538dbf5fedaa.png)

Lots of stuff searching for this error not related.  The way I was able to fix this was by editing a related library:

![image_code_change](https://user-images.githubusercontent.com/41549061/216796830-20a17223-3099-42f4-8d54-d1f95fa37881.png)

I just commented out the three highlighted lines and saved the file.  Obviously, this is not an elegant solution.  But, just to get people working, I thought for now this may be adequate.

This is the test code working now:

![image2](https://user-images.githubusercontent.com/41549061/216796916-d8a38ebd-14cb-4551-b423-cf2170c333b9.png)

Hope this helps someone or improves setup documentation.

**Comment by fernvenue** [comment]
Fixed in https://github.com/arc53/docsgpt/pull/13.
